### ISSUE TITLE
The Loss is nan

### ISSUE THREAD

**Issue by FFFFFhh** [Paranioar/SGRAF#21]
Hello thankyou so much for reviewing this issue and reply

When I run train.py ,the result's Loss is nan,what's the problem?

**Comment by Paranioar** [comment]
Could you please give me more details, including the versions of python and pytorch, etc.
My initial idea is that you may adopt too new version of pytorch > 2.0.0. Try torch==1.7.1?

**Comment by FFFFFhh** [comment]
Excuse me，sorry for replying too late.I'v already changed the version of python==3.6、PyTorch==1.7.1、cudatoolkit==11.0.3，but still have the problem that loss is nan。




&nbsp;


冯浩瀚
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Haiwen ***@***.***&gt;; 
发送时间: 2024年6月16日(星期天) 下午2:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Paranioar/SGRAF] The Loss is nan (Issue #21)














 
Could you please give me more details, including the versions of python and pytorch, etc.
 My initial idea is that you may adopt too new version of pytorch &gt; 2.0.0. Try torch==1.7.1?
